core/data/core.joke

```diff
@@ -2149,7 +2149,7 @@
 
 (def
   ^{:arglists '([])
-    :doc "Reads the next line from *in*."
+    :doc "Reads the next line from *in*. Returns nil if an error (such as EOF) is detected."
```

Comment:
If I read the docs correctly, `ReadLine` won't return an error on EOF: `No indication or error is given if the input ends without a final line end.`. Which means this comment is not accurate: `read-line` will return the input until EOF, not `nil`, and I think this is how Clojure works as well.